pontificia-universita-gregoriana-teologia.csl

```diff
@@ -0,0 +1,539 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" initialize-with-hyphen="true" page-range-format="expanded" demote-non-dropping-particle="sort-only" name-delimiter=" &#8211; " names-delimiter=" &#8211; " delimiter-precedes-last="always" et-al-min="4" et-al-use-first="1" name-form="long" initialize-with=". " sort-separator=", ">
+  <info>
+    <title>Pontificia Università Gregoriana - Teologia</title>
+    <title-short>PUGTeologia</title-short>
+    <id>http://www.zotero.org/styles/pontificia-universita-gregoriana-teologia</id>
+    <link href="http://www.zotero.org/styles/pontificia-universita-gregoriana-teologia" rel="self"/>
+    <link href="https://www.unigre.it/it/teologia/documenti/documenti-dellunita-accademica/norme-tipografiche/" rel="documentation"/>
+    <author>
+      <name>Pau Manent Bistué</name>
+      <email>pmb_manent@yahoo.com</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="theology"/>
+    <summary>PUG style</summary>
+    <updated>2022-01-17T17:27:15+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <!-- Aquí es defineixen les questions concretes dels idiomes -->
+  <locale>
+    <style-options punctuation-in-quote="false"/>
+    <terms>
+      <term name="et-al"> &#8211; al.</term>
+      <term name="ibid">ibid.</term>
+      <term name="in">in</term>
+      <term name="accessed">accesso</term>
+      <term name="editor">ed.</term>
+      <term name="editor" form="short">ed.</term>
+      <term name="page-range-delimiter">-</term>
+      <term name="volume">Vol. </term>
+      <!-- usamos "volume" para indicar el volumen en nota -->
+      <term name="ordinal-01"> vol.</term>
+      <!-- usamos "ordinal-01" para indicar el volumen en bibliografia (faltan terms!!) -->
```

Comment:
Can you explain the difference between what you want for the volume terms in the note versus bibliography? They look like the same word to me but different spaces? Hacking the ordinal term isn’t appropriate for that. What are you trying to do exactly?

Reply:
Solved with other terms and using in-line text formatting. Thanks for your advice

Reply:
You control the casing of terms with thr `text-case` attribute, not by using other terms